src/ansible_navigator/actions/settings.py

```diff
@@ -0,0 +1,201 @@
+"""The ``settings`` subcommand action."""
+
+import curses
+
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+
+from .._yaml import human_dump
+from ..action_base import ActionBase
+from ..action_defs import RunStdoutReturn
+from ..app_public import AppPublic
+from ..configuration_subsystem import transform_settings
+from ..steps import Step
+from ..ui_framework import CursesLinePart
+from ..ui_framework import CursesLines
+from ..ui_framework import Interaction
+from . import _actions as actions
+from . import run_action
+
+
+def filter_content_keys(obj: Dict[Any, Any]) -> Dict[Any, Any]:
+    """Filter out some keys when showing content.
+
+    :param obj: The object to be filtered
+    :return: The object with keys filtered out
+    """
+    return {k: v for k, v in obj.items() if not k.startswith("__")}
+
+
+def color_menu(colno: int, colname: str, entry: Dict[str, Any]) -> Tuple[int, int]:
+    # pylint: disable=unused-argument
+    """Color the menu.
+
+    :param colno: Column number
+    :param colname: Column name
+    :param entry: Column value
+    :return: Tuple int used to color the menu
+    """
+    if entry["is_default"]:
+        return 2, 0
+    return 3, 0
+
+
+def content_heading(obj: Any, screen_w: int) -> CursesLines:
```

Comment:
The same naming concern as https://github.com/ansible/ansible-navigator/pull/957/files#r810637593 applies here too.

Reply:
Same as above, this follows the current pattern:

```
src/ansible_navigator/actions/inventory.py:53:def content_heading(obj: Any, screen_w: int) -> Union[CursesLines, None]:
src/ansible_navigator/actions/config.py:47:def content_heading(obj: Any, screen_w: int) -> Union[CursesLines, None]:
src/ansible_navigator/actions/collections.py:48:def content_heading(obj: Any, screen_w: int) -> Union[CursesLines, None]:
src/ansible_navigator/actions/run.py:103:def content_heading(obj: Any, screen_w: int) -> Union[CursesLines, None]:```

Reply:
agree, room for improvement, but in a PR across all actions

Reply:
Resolving, since this is out of scope for this PR. Same as the above conversation.